_posts/2025-05-09-opentelemetry-zero-code-instrumentation.md

```diff
@@ -0,0 +1,537 @@
+---
+title: "Zero-code Instrumentation for OpenFaaS Functions with OpenTelemetry"
+description: "Learn how to capture telemetry data for OpenFaaS functions without modifying your code."
```

Comment:
It's not telemetry - as in user activity, it's traces.

"Capture traces from your functions to debug tricky issues, and to identify potential improvements."